### DISCUSSION TITLE
Domains stuck on "Generating SSL Certificate"

### DISCUSSION THREAD

**Discussion by dsaltares** [vercel/vercel#8516] [Help] [answered]
I have configured the `saltares.com` and `www.saltares.com` in my Vercel project to serve a Next.js statically exported site. After setting the DNS in my domain provider (namecheap), Vercel seems to be stuck on "Generating SSL Certificate".

I was migrating this site from Github pages. As it's been like that for over 1h, my site is currently inaccessible.

<img width="998" alt="image" src="https://user-images.githubusercontent.com/570314/188409267-4f491257-7ca8-42ec-8397-1e496b37fbb5.png">

I saw this discussion as well https://github.com/vercel/vercel/discussions/5348 but:
- I am not using Cloudflare
- Re-deploying has no effect
<img width="1299" alt="Screenshot 2022-09-05 at 10 44 58" src="https://user-images.githubusercontent.com/570314/188409949-c27529ba-132f-470e-b5e6-7e0bf88c3e6e.png">

I have also added a CAA record for lets encrypt.
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/570314/188413859-a7a32eec-bda9-4d00-8a19-d5b76e60cc65.png">

Any clues?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @dsaltares! It looks like you may have already tried this, but we have a related guide: [Why is my domain not automatically generating an SSL certificate?](https://vercel.com/guides/domain-not-generating-ssl-certificate). If you're still having problems, the next step is creating a ticket using the support form at [vercel.com/help](https://vercel.com/help). That way we can dig into your domain settings with you while keeping your account information safe out of this public discussion.
